common/djangoapps/entitlements/models.py

```diff
@@ -0,0 +1,112 @@
+from django.db import models
+from django.contrib.auth.models import User
+from course_modes.models import CourseMode
+
+
+class CourseEntitlement(models.Model):
+    """
+    Represents a Student's Entitlement to a Course Run for a given Course.
+    """
+
+    user = models.ForeignKey(User)
+
+    parent_course_uuid = models.UUIDField(primary_key=True)
```

Comment:
use of `parent_course` is confusing.  It is either course and course_run or parent_course and course.  But it cannot be both.  We need a shared language.

Reply:
Also, this should not be the primary key IMHO

Reply:
Agree, this should just be course.

Reply:
course_uuid it is